ui/actions/nuxeo-download-button.js

```diff
@@ -97,7 +101,7 @@ import '../nuxeo-button-styles.js';
 
         _label: {
           type: String,
-          computed: '_computeLabel(i18n)',
+          computed: '_computeLabel(document, i18n)',
```

Comment:
This should be _computeHoverLabel

Reply:
removed the extra param. function name is correct.